runtime/compiler/build/toolcfg/gnu-xlc/common.mk

```diff
@@ -108,6 +108,14 @@ endif
 
 ifeq ($(BUILD_CONFIG),prod)
     CX_DEFINES+=$(CX_DEFINES_PROD)
+    ifeq (,$(findstring ppc64le,$(PLATFORM)))
```

Comment:
@gita-omr please review this change, thank you!

Reply:
Approved, but may be we should add a comment in the make file why we need the NETHUNK option?

Reply:
Thank you @gita-omr 
I added a comment in the makefile and updated the commit message too.